records_mover/records/delimited/conversions.py

```diff
@@ -0,0 +1,80 @@
+from .types import HintCompression, HintEncoding, HintDateFormat, HintTimeOnlyFormat, HintQuoting
+import logging
+import csv
+from typing import Optional, Dict, Union
+from typing_extensions import Literal
+
+
+logger = logging.getLogger(__name__)
+
+
+pandas_compression_from_hint: Dict[HintCompression, Optional[str]] = {
+    'GZIP': 'gzip',
+    'BZIP': 'bz2',
+    None: None,
+}
+
+
+pandas_quoting_from_hint: Dict[HintQuoting, int] = {
+    'minimal': csv.QUOTE_MINIMAL,
+    'all': csv.QUOTE_ALL,
+    'nonnumeric': csv.QUOTE_NONNUMERIC,
+    None: csv.QUOTE_NONE
+}
+
+python_encoding_from_hint: Dict[Optional[HintEncoding], str] = {
```

Comment:
I split these into their own file and annotated them with some of the new types!